Sources/TensorFlow/Initializers.swift

```diff
@@ -499,3 +499,54 @@ public extension Tensor where Scalar: BinaryFloatingPoint,
         self = Tensor.glorot(fromStandardUniform: uniform, shape: shape)
     }
 }
+
+fileprivate extension Tensor where Scalar: BinaryFloatingPoint {
```

Comment:
It seems that this function is exactly the same as the one defined on line 452. Why do you need to defined this? Also, the `varianceScalingFactor` parameter is completely unused in the function body.

Reply:
@rxwei The function on line 452 does it by uniform distribution, this function does it by normal distribution, I read parameters could be overloaded in swift as long as the names are swift, Am i wrong?
 And `varianceScalingFactor` is something proposed in the above review, Just for better variable naming.

Reply:
> The function on line 452 does it by uniform distribution, this function does it by normal distribution,

The function is almost exactly the same as the one defined on line 452, character by character, except for the last line. Maybe there is a way to unify the identical logic to a separate method.

> I read parameters could be overloaded in swift as long as the names are swift, Am i wrong?

Yep, things can be overloaded and I'm not worrying about this.

> And `varianceScalingFactor` is something proposed in the above review, Just for better variable naming.

However, the point I brought up was that this variable is unused _at all_ in the body of this function, and compilation failed.

Reply:
@rxwei how should I move forward on this now? Primarily I'll fix the variable error.

Reply:
Please fix the variable error for now.

Reply:
@rxwei fixed the variable errors and other minor things, This passes locally now.